src/init.c

```diff
@@ -309,6 +311,9 @@ void attribute_visible R_init_datatable(DllInfo *info)
 
   initDTthreads();
   avoid_openmp_hang_within_fork();
+
+  R_RegisterCCallable("data.table", "CfifelseR", (DL_FUNC) &fifelseR);
```

Comment:
This is about opening these functions for packages `LinkingTo` `data.table`, right?

Reply:
such exports on C level should be documented somewhere, and in first place shouldn't happen unless we are sure that API is stable enough.

Reply:
Yes it is to make these functions available at C level to other packages. I can remove these lines, or can add a couple of lines in NEWS. I was just testing what was documented in the R extension manual.

Reply:
could we have those two new C exports removed from this PR?
lets have R fcase completed first, and also this PR merged first https://github.com/Rdatatable/data.table/pull/4084

Reply:
I think i did  all changes you requested. Please let me know if I missed some. Thank you.